includes/site-health.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+class PMProSiteHealth{
+
+	/**
+	 * Singleton Handling
+	 *
+	 * return $instance
+	 */
+	public static function init() {
+
+		if ( ! is_object( self::$instance ) ) {
+			self::$instance = new PMProSiteHealth();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Setup and handle Debug Information hook
+	 *
+	 */
+	public function __construct(){
+
+		add_filter( 'debug_information', array( $this, 'debug_information' ) );
+
+	}
+
+	/**
+	 * Handles debug information
+	 * @param  [array] $info [Existing Site Health data]
+	 * @return [array]       [New Site Health Data Added]
```

Comment:
```suggestion
	 * Add our debug information to Site Health.
	 *
	 * @since TBD
	 *
	 * @param array $info The existing Site Health data.
	 *
	 * @return array The Site Health data with ours added.
```

* We should use normal sentences with `.` at the end.
* Let's also add `@since` to other methods.